dex/elrond_dex_pair/src/lib.rs

```diff
@@ -357,15 +362,15 @@ pub trait Pair:
     }
 
     #[payable("*")]
-    #[endpoint(swapTokensFixedInput)]
-    fn swap_tokens_fixed_input(
+    #[endpoint]
+    fn swapTokensFixedInput(
```

Comment:
Any reason for the rename? Rust standard requires functions to have snake_case naming.

Reply:
After long debugging sessions while rust framework was not really stable and released, I've encountered the situation where an exec on dest would not work if the function was defined with snake_case and the endpoint with camelCase. Probably now it's fine, will test and update

Reply:
done